빌드 실패하는 현상 해결

```
$  ./gradlew build 
> Task :bootJar FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':bootJar'.
> Error while evaluating property 'mainClass' of task ':bootJar'
   > Failed to calculate the value of task ':bootJar' property 'mainClass'.
      > Main class name has not been configured and it could not be resolved

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 1s
2 actionable tasks: 2 executed

```

--- a/api/build.gradle
+++ b/api/build.gradle
@@ -1,6 +1,8 @@
-import org.springframework.boot.gradle.tasks.bundling.BootJar
-
 dependencies {
     api(project(":domain"))
     implementation("org.springframework.boot:spring-boot-starter-web")
+}
+
+jar {
+    enabled = false
 }

--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -1,4 +1,5 @@
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+import org.springframework.boot.gradle.tasks.bundling.BootJar
 
 plugins {
 	val bootVer = "2.5.3"
@@ -68,4 +69,8 @@ subprojects {
 	tasks.withType<Test> {
 		useJUnitPlatform()
 	}
+}
+
+tasks.withType<BootJar> {
+	enabled = false
 }

--- a/domain/build.gradle
+++ b/domain/build.gradle
@@ -4,3 +4,6 @@ dependencies {
     api("org.springframework.boot:spring-boot-starter-data-jpa")
 }
 
+tasks.bootJar {
+    enabled = false
+}

